src/Simplex/Chat.hs

```diff
@@ -338,8 +354,12 @@ chatCommandP =
     contactRef = safeDecodeUtf8 <$> (B.cons <$> A.satisfy refChar <*> A.takeTill (== ' '))
     refChar c = c > ' ' && c /= '#' && c /= '@'
     groupRef = contactRef
+    groupProfile = do
+      gRef <- groupRef
+      gName <- safeDecodeUtf8 <$> (A.space *> A.takeByteString) <|> pure ""
+      pure GroupProfile {groupRef = gRef, displayName = if T.null gName then gRef else gName}
     memberRole =
-      ("owner" $> GROwner)
-        <|> ("admin" $> GRAdmin)
-        <|> ("normal" $> GRMember)
-        <?> "memberRole"
+      (" owner" $> GROwner)
+        <|> (" admin" $> GRAdmin)
+        <|> (" normal" $> GRMember)
+        <|> pure GRMember
```

Comment:
spaces moved here because role is optional?

Reply:
that's the idea